automated/android/noninteractive-tradefed/tradefed.sh

```diff
@@ -123,7 +123,13 @@ file_name=$(basename "${TEST_URL}")
 if echo "${TEST_URL}" | grep "^http://lkft-cache.lkftlab/" ; then
     NO_PROXY=.lkftlab wget -S --progress=dot:giga "${TEST_URL}" -O "${file_name}"
 elif echo "${TEST_URL}" | grep "^http" ; then
-    wget -S --progress=dot:giga "${TEST_URL}" -O "${file_name}"
+    ### using kisscache to download the file, based on the following change:
+    ###    https://gitlab.com/lava/lava/-/merge_requests/2734
+    if [ -n "${HTTP_CACHE}" ]; then
```

Comment:
The setting is exposed directly. It is this format: `https://cache.lavasoftware.org/api/v1/fetch/?url=%s`. I think you need to replace the `%s` with `TEST_URL` here.

Reply:
Thanks for pointing it out! updated to remove %s first.